llama_finetuning.py

```diff
@@ -62,8 +63,10 @@
 from torch.optim.lr_scheduler import StepLR
 from pkg_resources import packaging
 import torch
+import torch.nn as nn
 import torch.cuda.nccl as nccl
 import torch.distributed as dist
+from torch.distributed.fsdp._common_utils import _is_fsdp_flattened
```

Comment:
Is this being used anywhere in the code? Will be good to remove the unused imports, esp for the "_" methods which are internal and may change in future